ui/components/ui/actionable-message/actionable-message.js

```diff
@@ -50,7 +54,21 @@ export default function ActionableMessage({
           wrapperClassName="actionable-message__info-tooltip-wrapper"
         />
       )}
-      <div className="actionable-message__message">{message}</div>
+      <div
+        className={
+          learnMore === ''
+            ? 'actionable-message__message'
+            : 'actionable-message__message-grey'
+        }
+      >
+        {message}
+        {learnMore === '' ? null : (
```

Comment:
The "message" prop can be a react node, so i don't see why we need to have this very specific prop and implementation here that will likely not be reused. Instead...

```js
        <ActionableMessage
          type="warning"
          message={<>
            {t('onlyInteractWith')}
	          <a href={learnMore} style={{ color: '#037DD6' }}>
              {' '}
              {t('endOfFlowMessage9')}
            </a>
         </>}
          iconFillColor="#f8c000"
          useIcon
          withRightButton
          learnMore="https://metamask.io/"
        />
```

something like this would work. Although I think you should use a classname instead of passing the style attribute to the a tag. As for the class name you're appending to the div 'actional-message__message-grey' i would prefer we simply modify the actionable-message__message from the new page's CSS.

Reply:
Fixed